oriental_motor_actuator:
  port: '/dev/'
  baud_rate: 115200
  parity: 'E'
  data_bit: 8
  stop_bit: 1